src/julia.h

```diff
@@ -1799,6 +1799,11 @@ typedef struct _jl_task_t {
     uint8_t sticky; // record whether this Task can be migrated to a new thread
 
 // hidden state:
+    // id of owning thread - does not need to be defined until the task runs
+    int16_t tid;
+    // multiqueue priority
+    int16_t prio;
+
     jl_ucontext_t ctx; // saved thread state
```

Comment:
We could potentially store this inside the stkbuf object too, if that's helpful for making it cheaper